Networking/Extensions/AlamofireExtensions.swift

```diff
@@ -38,7 +38,7 @@ internal extension Alamofire.DataRequest {
     
     func response() -> SignalProducer<ResponseType, ResponseError> {
         return SignalProducer { observer, disposable in
-            disposable.add { self.task?.cancel() }
+            disposable.observeEnded { self.task?.cancel() }
```

Comment:
Are you sure? It sounds weird to me. Do you have a link or something with this?

Reply:
This was a suggestion of Xcode, `'add' has been renamed to 'observeEnded'`: https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/Sources/Deprecations%2BRemovals.swift#L21